mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/Algo.scala

```diff
@@ -17,14 +17,12 @@
 
 package org.apache.spark.mllib.tree.configuration
 
-import org.apache.spark.annotation.{Experimental, Since}
+import org.apache.spark.annotation.Since
 
 /**
- * :: Experimental ::
  * Enum to select the algorithm for the decision tree
  */
 @Since("1.0.0")
-@Experimental
```

Comment:
Tree configurations should stay experimental. Very likely we should deprecate them. The recommended way is to use flattened parameters.